### PR TITLE
Setup Client

### PR DESCRIPTION
**Pull request by johanbounce**
Initial setup for client.
Configured:
• Semaphore
• Netlify
• Coveralls
• Cypress
• Jest and Enzyme
• Clean up files we don't  need

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
+// axios.defaults.baseURL = "https://localhost:3000/";
+
```
Comment (faraznaeem) [inline src/index.js]: delete comments

**Comment by johanbounce** [comment]
Please state if something is missing. We can't find anything missing for the initial setup.

**Review by faraznaeem — CHANGES_REQUESTED** [review]
Dont understand what this PR does, please clarify

File: src/index.js (hunk 1 above)
Comment (johanbounce) [reply src/index.js]: Basically forking and cloning the repo. Setting up (also making sure it's set up) all the tools such as Semaphore, Coveralls etc. that was included as user stories by oliver in the PT.

Comment (johanbounce) [reply src/index.js]: Comment is deleted

**Pull request closed without merging**